### PR TITLE
UMIP-14

### PR DESCRIPTION
**Pull request by nicholaspai**
UMIP to deploy a new EMPCreator contract supporting an updated EMP contract

### CODE REVIEW

File: UMIPs/umip-14.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+This UMIP was motivated by a combination of user comments and economic attack surface considerations.
+
+## Technical Specification
+To accomplish this upgrade, a new financial contract template must be deployed.
+
+After deployment, this new `ExpiringMutltiPartyCreator` contract should be approved as a ContractCreator in the Registry.
+
+Note: the voters only vote for the approval -- the deployment has no effect on the protocol until added as a ContractCreator.
+
+The specific changes in detail comprise:
```
Comment (chrismaree) [inline UMIPs/umip-14.md:28]: Formatting nit: consider making this bold?
```suggestion
**The specific changes in detail comprise:**
```

File: UMIPs/umip-14.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+## Headers
+| UMIP-14    |                                                                                                                                          |
+|------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| UMIP Title | Approve Updated EMP Financial Contract Template              |
+| Authors    | Nick Pai (nick@umaproject.org) |
+| Status     | Draft                                                                                                                                    |
+| Created    | September 16, 2020                                                                                                                           |
+
+## Summary (2-5 sentences)
```
Comment (chrismaree) [inline UMIPs/umip-14.md:9]: I think you are missing the changes made in PR [1553](https://github.com/UMAprotocol/protocol/pull/1753) which added new events in the expiring multi-party template. Would you agree this should be added?

Comment (chrismaree) [inline UMIPs/umip-14.md:9]:
```suggestion
## Summary
```

Comment (nicholaspai) [reply UMIPs/umip-14.md:9]: These were added in this [UMIP](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-9.md#implementation) actually

**Review by daywiss — COMMENTED** [review]
Seems really good to me. I will defer to the more experienced guys to approve

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by chrismaree — APPROVED** [review]
Looks great! Thanks for the fast turn around time. Super excited about these features.

**Pull request merged**